README.md

```diff
@@ -166,40 +170,248 @@ Remove a key (and subsequent value) from this object. For example, to delete the
 "year" property from our car:
 
 ``` handlebars
-<button onclick={{car.delete "year"}}>
+<button onclick={{action car.delete "year"}}>
   Remove Year
 </button>
 {{!clicking will result in {make: 'Ford', model: 'Mustang'}}}
 ```
 
 #### `put(key, value)`
-#### `set(value)`
+
+Add property with a given name and value to the object. It will update the property with the given value if it already exists.
+
+```handlebars
+<button onclick={{action car.put "color" "blue"}}>
+  Add color
+</button>
+```
+
+#### `set(obj)`
```

Comment:
In order for the link to work, I think this needs to match the link above (minus any special characters). So in this case the link is `[set(object)](#setvalue)` and the heading is `set(obj)`

A good match might be `[set(object)](#setobject)`